### PR TITLE
Add Code Review to Contributing.md #63

### LINKED ISSUE #63: Add Code Review to Contributing.md
- Add a "Code Reviews" section on CONTRIBUTING.md
- Put the following under the header:
> If you can, please look at open PRs and review them. Give feedback and help us merge these PRs much faster! If you don't know how, Github has some great [information on how to review a Pull Request](https://help.github.com/articles/about-pull-request-reviews/).
- Link the header in the Contributing.md Table of Contents
- Add a link to the README.md file: [Review Pull Requests]("CONTRIBUTING.md#Code-Reviews") under "Contributing Section"

### PR DESCRIPTION
**Pull request by derekneuland**
# Fixes #63 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- 
- 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/ruby-http-client?pullRequest=64) <br/>All committers have signed the CLA.

**Review by kplimack — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -190,5 +191,8 @@ Please run your code through:
 
 7. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/)
     with a clear title and description against the `master` branch. All tests must be passing before we will review the PR.
+    
+## Code Reviews<a name="code-reviews"></a>
```
Comment (kplimack) [inline CONTRIBUTING.md:195]: you do not need to insert an HTML anchor tag, github makes anchors automatically for every heading you make, inserting dashes in lieu of spaces

**Review by kplimack — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kplimack) [reply CONTRIBUTING.md:195]: the link would look like:
```
[Code Reviews](#code-reviews)
```
so that doesnt change.

**Comment by childish-sambino** [comment]
Duplicates #100

**Pull request closed without merging**